docs/content/en/docs/packages/cluster-autoscaler/addclauto.md

```diff
@@ -120,47 +104,12 @@ eksctl anywhere apply package -f cluster-autoscaler.yaml
 
 Cluster Autoscaler will automatically be upgraded when a new bundle is activated.
 
+It is possible to change the autoscaling configuration of a worker node group by updating the `WorkerNodeGroup.autoscalingConfiguration` either via the Full Lifecycle Controller or running an upgrade through the CLI.
```

Comment:
Can we say "It is possible to change the autoscaling configuration of a worker node group by updating the `WorkerNodeGroup.autoscalingConfiguration` in your cluster specification" 

I want to remove "Full Lifecycle Controller" vs "CLI" language because we don't explain that well anywhere else in the docs.

Reply:
This should go in the Update section.